docs/_static/custom.js

```diff
@@ -39,20 +41,36 @@ function getRootAttributeToggle(attributeName, valueName) {
   return toggleRootAttribute;
 }
 
+function toggleClassicAPIRef(add) {
+  if (apiReference) {
+    var event = new CustomEvent('APIRefStyle', { detail: { classic: add } });
```

Comment:
```suggestion
    let event = new CustomEvent('APIRefStyle', { detail: { classic: add } });
```

What is this custom event for anyway?